hawkbit-repository/hawkbit-repository-core/src/test/java/org/eclipse/hawkbit/event/BusProtoStuffMessageConverterTest.java

```diff
@@ -65,12 +65,11 @@ public void successfullySerializeAndDeserializeEvent() {
     @Description("Verifies that a MessageConversationException is thrown on missing event-type information encoding")
     public void missingEventTypeMappingThrowsMessageConversationException() {
         final DummyRemoteEntityEvent dummyEvent = new DummyRemoteEntityEvent(targetMock, "applicationId");
-        try {
-            underTest.convertToInternal(dummyEvent, new MessageHeaders(new HashMap<>()), null);
-            Assertions.fail("Missing MessageConversationException for un-defined event-type");
-        } catch (final MessageConversionException e) {
-            // expected exception
-        }
+        final MessageHeaders messageHeaders = new MessageHeaders(new HashMap<>());
+
+        org.junit.jupiter.api.Assertions.assertThrows(MessageConversionException.class,
+                () -> underTest.convertToInternal(dummyEvent, messageHeaders, null),
```

Comment:
I guess you could use the assertJ here as it is already imported and present. Moreover, there are currently a lot of inconsistencies whether we use Junit5 Jupiter exception assertions or the ones provided by assertJ. I don't expect we fix all of the inconsistency issues all together, and definitely not in the scope of this PR, however I would try to at least keep the consistency within the test class (e.g. see PreAuthTokenSourceTrustAuthenticationProviderTest.java )

Reply:
Yes you were right. I just replaced all  Junit5 Jupiter exception assertions by the one from assertJ since they offer more flexibility.